grass/kernel.c

```diff
@@ -114,6 +120,11 @@ static void proc_yield() {
     earth->mmu_switch(curr_pid);
     earth->mmu_flush_cache();
 
+    /* Student's code goes here (preemptive scheduler)
+     * Update the new process's scheduling information. */
```

Comment:
Say "Measure and record scheduling metrics for the next process." And move this comment to line 105, i.e., right after the for loop where `next_idx` represents the next process.